amundsen_application/static/css/_icons.scss

```diff
@@ -196,6 +196,11 @@ img.icon {
     -webkit-mask-image: url('/static/images/icons/More.svg');
     mask-image: url('/static/images/icons/More.svg');
   }
+
+  &.icon-check {
+    -webkit-mask-image: url('/static/images/icons/check.svg');
+    mask-image: url('/static/images/icons/check.svg');
+  }
```

Comment:
I don't recommend to use this approach for new icons, as it is not supported on Firefox and other browsers and frankly, is less than ideal.

I would try to follow the pattern at the top, using the iconBackgrounds mixin